R/spinMyR.R

```diff
@@ -45,6 +45,8 @@
 #' \code{DATASET_NAME}, then you can use
 #' \code{params = list('DATASET_NAME' = 'oct10dat')}
 #' @param warn If TRUE, then show warnings (recommended to keep this on)
+#' @param keepRmd,keepMd Should intermediate \code{Rmd} or \code{md} files be
+#'   kept (\code{TRUE}) or deleted (\code{FALSE})?
 #' @return The path to the output (invisibly).
 #' @section Possible future improvements:
```

Comment:
Could you remove this "future improvements" section since this is essentially what you just added